src/io.ml

```diff
@@ -2,11 +2,94 @@ open Import
 
 module P = Pervasives
 
+module W32_io = struct
```

Comment:
Just for consistency with other naming, please could this `Win32` rather than `W32`?